tests/vsphere_test.go

```diff
@@ -43,6 +44,24 @@ const (
 
 var templateID string
 
+// This versioning scheme that currently seems to be in place for template build numbers.
+var buildNumberRegex = regexp.MustCompile(`[bB]?(?P<version>\d+)`)
+
+func getBuildNumberFromString(build string) int {
```

Comment:
I'd call it `extractBuildNumber`, which is a bit shorter - but that's a big taste-only thing, am fine with this name too :)

Reply:
No I was very unhappy with this name anyways, just didn't come up with a better one so far. So I will change this